cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/vsphere/vsphere-mc-common/VsphereClusterSettingsStep.tsx

```diff
@@ -130,6 +136,24 @@ export function VsphereClusterSettingsStep(props: Partial<StepProps>) {
         setSelectedInstanceTypeId(instanceType);
     };
 
+    const handleRefresh = async (event: MouseEvent<HTMLAnchorElement>) => {
+        event.preventDefault();
+        fetchOsImages();
+    };
+
+    const fetchOsImages = async () => {
+        try {
+            setOsImagesLoading(true);
+            await VsphereService.getVSphereOsImages(vsphereState[STORE_SECTION_FORM][VSPHERE_FIELDS.DATACENTER]).then((fetchedOsImages) => {
+                setOsImages(fetchedOsImages);
+            });
+        } catch (e: any) {
+            console.log(`Unabled to get ec2 key pair: ${e}`);
```

Comment:
error log should say "Unable to retrieve OS Images"

Reply:
Right sir.